src/api/todos.ts

```diff
@@ -5,4 +5,18 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const createTodo = (userId: number, title: string) => {
+  return client.post<Todo>('/todos', {
+    userId,
+    title,
+    completed: false,
+  });
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const updateTodo = (todoId: number, data: any) => {
```

Comment:
try to avoid of using `any` in types. especially that VSC or Webshtorm gives you a tip about global object types)